integrations/torchvision/tutorials/sparse-transfer-learning.md

```diff
@@ -0,0 +1,168 @@
+# Sparsifying Image Classification Models with the CLI
+
+This page explains how to fine-tune a pre-sparsified Torchvision image classification models like ResNet-50 with SparseML's CLI.
```

Comment:
This page explains how to fine-tune ~~a~~ pre-sparsified Torchvision image classification models like ResNet-50 with SparseML's CLI.